src/main/java/frc/robot/constants/StorageIndexConstants.java

```diff
@@ -15,5 +15,9 @@ public class StorageIndexConstants {
     public static final double ejectAmpBackSpeed = 0;
     public static final double ejectTrapSpeed = 0;
 
+    public static final double intakeTimeout = 0;
```

Comment:
We have the beam break sensor for this one. We need an ejectShooterTimeout instead.